ember-server/shared/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala

```diff
@@ -210,8 +210,18 @@ final class EmberServerBuilder[F[_]: Async: Network] private (
   ): EmberServerBuilder[F] =
     copy(requestLineParseErrorHandler = requestLineParseErrorHandler)
 
+  private val verifyTimeoutRelations: F[Unit] =
+    logger
+      .warn(
+        s"requestHeaderReceiveTimeout ($requestHeaderReceiveTimeout) is >= idleTimeout ($idleTimeout). " +
+          s"It is recommended to configure requestHeaderReceiveTimeout < idleTimeout, " +
+          s"otherwise timeout responses won't be delivered to clients."
```

Comment:
Sorry, what do you mean by this? It's not immediately obvious to me how the value of this setting relates to clients receiving timeout responses.

https://github.com/http4s/http4s/blob/8744d6c4f62dfac1a2f55f7a928222325dc73eaa/ember-server/shared/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala#L365-L373

Reply:
Sorry my memory's a little hazy on the details... I think the message was broadly copied from [Blaze](https://github.com/http4s/blaze/blob/75f4a935c2426dcee8dea9d780b6045a66efedf2/blaze-server/src/main/scala/org/http4s/blaze/server/BlazeServerBuilder.scala#L459).

Looking again, it looks like `requestHeaderReceiveTimeout` is different to the `responseHeaderTimeout` in Blaze (which was closer to the equivalent `timeout` on the client) and this relation doesn't apply any more

It's the client timeout (mis)configuration that has bitten me in the past - I'll remove the server configuration warnings now